### PR TITLE
Adds new repository for BaSyx AAS Web UI

### PR DESCRIPTION
**Pull request by aaronzi**
We've decided to move the BaSyx AAS Web UI into it's own repository. Over the last few years it had grown considerably in size and got a lot of attention as well as pulls from dockerhub. In order to separate it from the rest of the BaSyx applications and to improve development productivity, moving it to it's own repository seems to be beneficial.

### CODE REVIEW

**Comment by netomi** [comment]
LGTM, once a PL approves you can merge using `/otterdog merge`

**Review by FrankSchnicke — APPROVED** [review]

**Review by netomi — APPROVED** [review]

**Pull request merged**
